app/models/consultation.rb

```diff
@@ -184,6 +184,10 @@ def locale_can_be_changed?
     true
   end
 
+  def string_for_slug
+    title
```

Comment:
I wasn't sure why this change was made, until I read it in the PR description:

> Ensure that a non-:en localed Consultation will use their title for their slug (logic otherwise exists to use the id)

Could this info be added to the commit too?

Reply:
good point - this was included in the "template" PR but neglected here.